### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jose5918**
Updating the readme with information that was moved from this pull request https://github.com/kubeflow/community/pull/45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,14 +125,14 @@ configured for the repository (see these [instructions](#prow-setup) for info on
   * The first step in the workflow should checkout the code using [checkout.sh](https://github.com/kubeflow/testing/tree/master/images/checkout.sh)
   * Code should be checked out to a shared NFS volume to make it accessible to subsequent steps
 1. Create a container to use with the Prow job
-  * For an example look at the [tensorflow/k8s](https://github.com/tensorflow/k8s/tree/master/test/test-infra) repository
+  * For an example look at the [kubeflow/testing](https://github.com/kubeflow/testing/blob/master/images/Dockerfile) repository
```
Comment (ankushagarwal) [inline README.md]: For an example look at the [kubeflow/testing](https://github.com/kubeflow/testing/blob/master/images/Dockerfile) ~~repository~~ _Dockerfile_

File: README.md [hunk 2]
```diff
@@ -125,14 +125,14 @@ configured for the repository (see these [instructions](#prow-setup) for info on
...
   * Image should be based on `gcr.io/mlkube-testing/test-worker`
```
Comment (ankushagarwal) [inline README.md]: Can you update this to gcr.io/kubeflow-ci/test-worker

**Comment by jose5918** [comment]
@ankushagarwal PTAL

**Comment by jlewi** [comment]
@jose5918 Looks like lint issues.

**Comment by ankushagarwal** [comment]
@jose5918 If you rebase this PR tests should pass

**Comment by jose5918** [comment]
@ankushagarwal rebased

**Comment by ankushagarwal** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**